Meta/serenity_gdb.py

```diff
@@ -186,10 +190,10 @@ def __init__(self, val):
         self.val = val
 
     def to_string(self):
-        if int(self.val["m_length"]) == 0:
-            return '""'
-        else:
-            return self.val["m_characters"].string(length=self.val["m_length"])
+        return self.val["m_characters"].string(length=self.val["m_length"])
```

Comment:
Mention StringView in commit description too.

Actually, could you do one commit per thing?
* One for adding IntrusiveList
* One for fixing HashMap (add some more words on what exactly it fixes, see https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md `Don't Use weasel-words like "refactor" or "fix" to avoid explaining what's being changed.`)
* One for StringView, saying what exactly is changing here (I think this here is meant to be a no-op)